app/views/posts/_form.html.erb

```diff
@@ -6,6 +6,21 @@
     <%= form.text_field :body %>
   </div>
 
+  <div>
+    <%= form.label :images, style: "display: block" %>
+    <%= form.file_field :images, multiple: true %>
+    <% @post.images.each do |image| %>
```

Comment:
You can check ifr the images exist here with this 
<% if @post.images.any? %> so that if they arent there nothing is disolayed

Reply:
Done!